classes/Tools.php

```diff
@@ -2110,6 +2142,15 @@ public static function refreshCACertFile()
         }
     }
 
+    /**
+     * @param string $url
+     * @param int $curl_timeout
+     * @param array $opts
+     *
+     * @return bool|mixed
```

Comment:
```suggestion
     * @return bool|string
```
Set CURLOPT_RETURNTRANSFER to TRUE to return the transfer as a string of the return value of curl_exec() instead of outputting it out directly.

Reply:
Thanks I apply it